### DISCUSSION TITLE
Please add back Advanced Vertical Zooming

### DISCUSSION THREAD

**Discussion by micsthepick** [audacity/audacity#6757] [Development]
### Your idea

Add back vertical zooming.
Vertical zooming used to be a feature that allowed you to zoom to a specific range by click dragging two bounds in the vertical bar. This really should still be configurable, and not deprecated.

### Problem to be solved

finer grained zooming control and better accesibility for users without a scroll wheel or motor skill impairments.

### Prior art

Older audacity versions worked quite well

### Additional context

Please Upvote this below if you think it deserves attention and should be reintroduced!

**Comment by mrbumpy409** [comment]
I use this feature quite a bit in my sound design work. I'm really frustrated that the functionality is just entirely gone. I need to be able to see only a certain vertical range of the waveform at times, and the advanced vertical zoom was perfect for this. Now I'll have to find a way to have an older Audacity version around just for this task.

**Reply by pljones** [reply]
Yep, it was _really_ useful for precise editing of clipped waves or glitches, too.

(It's possible with weird key combinations eventually to zoom in, but the old way was done in a quick "snap" to where you wanted to go.)

**Comment by Kurtsley** [comment]
Do you mean like the feature that exists in the spectrogram view? Dragging on the vertical ruler will zoom in to the selected area. If that is correct, I agree it would be a good feature for the waveform view.

**Reply by pljones** [reply]
You used to be able to select a decibel range in the scale on the left of the waveform to zoom into that range.  Not quite the same as the spectrogram zoom (which is also nice).

**Comment by Njsmooth** [comment]
I agree. I do stuff that requires me to vertical zoom into specific spots. Previously, I could achieve this by scrolling with my mouse cursor focused on the spot I want to zoom in to. Now I have to zoom, scroll, zoom again, scroll again, rinse and repeat, because if I zoom in too far, scrolling takes ages. I don't see why the config was removed, I don't think it was a hinderance or issue in any capacity
